src/Application/Service/Property/PropertyViewRequest.spec.js

```diff
@@ -0,0 +1,66 @@
+import { builder } from '@Application/Service/Property/PropertyViewRequest';
```

Comment:
is it necessary to create a file like this one to specify every single request?

Reply:
What? This is a unit test

Reply:
Ah cool! Sorry my bad, we can move it to a tests folder outside src folder 😄

Reply:
In JavaScript is very common to have tests besides source code or under a `__tests__` sub-folder in the same path. Personally, I like having them all together and moving all toghether in case of refactor.

Reply:
Not sure if it is a good practise to mix source code with tests. In PHP is very common to have it in a different folder keeping same structure.